sgpt/app.py

```diff
@@ -130,8 +130,8 @@ def main(
         prompt = get_edited_prompt()
 
     api_host = cfg.get("OPENAI_API_HOST")
-    api_key = cfg.get("OPENAI_API_KEY")
-    client = OpenAIClient(api_host, api_key)
+    __api_key = cfg.get("OPENAI_API_KEY")
```

Comment:
While it might work as "private" property in `OpenAIClient` and `Config` classes, it doesn't works here, since it is a function, I can still see the `__api_key` in tracebacks.

Reply:
This is fixed in Rich version 13.1.0

Reply:
Updated the dependency list in `pyproject.toml` to reflect this: [380bd61](https://github.com/TheR1D/shell_gpt/pull/221/commits/380bd612ce59ded923c7d38502f09bbbf9668c09)